docs/docs/charts/bar.mdx

```diff
@@ -82,15 +82,15 @@ the color of the bars is generally set this way.
 | [`base`](#general) | `number` | Yes | Yes |
 | [`borderColor`](#styling) | [`Color`](../general/colors.md) | Yes | Yes | `'rgba(0, 0, 0, 0.1)'`
 | [`borderSkipped`](#borderskipped) | `string` | Yes | Yes | `'start'`
-| [`borderWidth`](#borderwidth) | <code>number&#124;object</code> | Yes | Yes | `0`
-| [`borderRadius`](#borderradius) | <code>number&#124;object</code> | Yes | Yes | `0`
-| [`clip`](#general) | <code>number&#124;object</code> | - | - | `undefined`
+| [`borderWidth`](#borderwidth) | <code>`number`&#124;`object`</code> | Yes | Yes | `0`
+| [`borderRadius`](#borderradius) | <code>`number`&#124;`object`</code> | Yes | Yes | `0`
+| [`clip`](#general) | <code>`number`&#124;`object`</code> | - | - | `undefined`
```

Comment:
You can use `\|` instead of the `&#124;` and remo the `<code>` tags. Cleaner that way.

Reply:
I had the doubt because some files were using the tags, other what you suggested. I have chosen the wrong one...
I'm doing..

Reply:
I only prefer this because:
![image](https://user-images.githubusercontent.com/27971921/102651078-8b0f9c80-4174-11eb-9c08-476ef5765b95.png)

Reply:
Fine! In this case, I have also to remove `<sup>` tag that I have added in another page.

Reply:
> Fine! In this case, I have also to remove `<sup>` tag that I have added in another page.

I have seen that <sup> is used also in another page therefore I will leave it.